scripts/mbedtls_dev/bignum_core.py

```diff
@@ -759,12 +759,23 @@ class BignumCoreExpMod(BignumCoreTarget, bignum_common.ModOperationCommon):
     """Test cases for bignum core exponentiation."""
     symbol = "^"
     test_function = "mpi_core_exp_mod"
-    test_name = "Core modular exponentiation"
+    test_name = "Core modular exponentiation (Mongtomery form only)"
     input_style = "fixed"
 
+    def arguments(self) -> List[str]:
+        # Input 'a' has to be given in Montgomery form
+        mont_a = (self.int_a * self.r) % self.int_n
```

Comment:
Please make methods to convert to/from Montgomery form.
```suggestion
        mont_a = self.to_montgomery(self.int_a)
```
And likewise in the result.

Reply:
On that note, conversion is already [done as such](https://github.com/Mbed-TLS/mbedtls/blob/development/scripts/mbedtls_dev/bignum_mod_raw.py#L95) in the tests for `BignumModRawConvertFromMont` and `BignumModRawConverttoMont`. 

I think we can allow the quick and easy transformation here, and create a new pr to add the conversion on a higher level if that is expected to be used by many tests.

It could be an instance method or an attribute.

Reply:
It's okay, I've separated them out and used the new functions in `BignumModRawConvertFromMont` and `BignumModRawConverttoMont`